simulators/ethereum/engine/Dockerfile

```diff
@@ -1,11 +1,6 @@
 # Generate the ethash verification caches.
 # Use a static version because this will never need to be updated.
 FROM ethereum/client-go:v1.10.20 AS geth
-RUN \
- /usr/local/bin/geth makecache  1     /ethash && \
- /usr/local/bin/geth makecache  30000 /ethash && \
- /usr/local/bin/geth makedag    1     /ethash && \
- /usr/local/bin/geth makedag    30000 /ethash
```

Comment:
I believe we can skip the line above too, and also L24 `COPY --from=geth /ethash /ethash`